src/curve/constant-product/ConstantProductPair.sol

```diff
@@ -1,5 +1,4 @@
-// TODO: Can we reduce the nesting by deleting the parent dir?
```

Comment:
I think the nesting makes it clearer which contract is for which curve. But don't have a super strong opinion of this tho

Reply:
Yeah lets leave as is, this was before we split out StablePair from StableMintBurn